README.md

```diff
@@ -40,3 +109,55 @@ var link = sc2.sign('transfer', {
 console.log(link);
 // => https://v2.steemconnect.com/sign/transfer?to=fabien&amount=1.000%20STEEM&memo=Hello%20World!
 ```
+
+Note that the URL generated by the sign() method does not contain a redirect URI parameter, so the user will not be redirected back to your app once the transaction is completed.
+
+### Logout
+The revokeToken() method will log the current user out of your application by revoking the access token provided to your app for that user: 
+```
+sc2.revokeToken(function (err, res) {
+  console.log(err, res)
+});
+```
+
+### Re-Steem
```

Comment:
It would be better to use the same title than the name of method "Reblog"

Reply:
Fixed.